conan/solo5/0.4.1/conanfile.py

```diff
@@ -11,7 +11,7 @@ class Solo5Conan(ConanFile):
 
     def source(self):
         repo = tools.Git(folder = self.name)
-        repo.clone(self.url + ".git",branch="v{}".format(self.version))
+        repo.clone(self.url + ".git",branch="master")
```

Comment:
May be point to the latest solo5 commit instead by providing the commit id? We might not want their newer commits to break our tests

Reply:
We don't actually use that path anymore.. The solo5 package is being built from another repo called "conan", so it would have to be changed there